### PR TITLE
Added Fake Samples and Test for Nsynth Dataset

### PR DESCRIPTION
**Pull request by captain-pool**
#154 @cyfra

### CODE REVIEW

File: tensorflow_datasets/audio/commonvoice.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+"""
```
Comment (cyfra) [inline tensorflow_datasets/audio/commonvoice.py]: you seem to be adding commonvoice - is that intended ?

Comment (captain-pool) [reply tensorflow_datasets/audio/commonvoice.py]: @cyfra I mistakenly committed the code to the same branch on which I was working for commonvoice. This resulted in appending of the Commit to the PR I made for commonvoice. So, I performed a git reset --soft, created a new branch for this issue and force-pushed. This branch creation step, appended all the commonvoice commits along with the Nsynth Commit.
Can you suggest me a way to fix this?

Comment (cyfra) [reply tensorflow_datasets/audio/commonvoice.py]: There are couple different ways to fix it (and plenty of git documentation if you want to know the details).

The simplest one I'd suggest is:
- create a new branch from MASTER (so it doesn't contain any of your changes)
- use "git cherry-pick"  to import the changes that you need
- push this branch to github.

Comment (captain-pool) [reply tensorflow_datasets/audio/commonvoice.py]: Sure. Then It would be better that I close this PR.

**Pull request closed without merging**